test/rpc/rpc-responseSpec.ts

```diff
@@ -0,0 +1,138 @@
+import { expect } from 'chai'
+import * as sinon from 'sinon'
+import { getServicesMock } from '../mocks'
+import { EVENT } from '../../src/constants'
+import { TOPIC, RPC_ACTIONS, RPCMessage } from '../../binary-protocol/src/message-constants'
+
+import { DefaultOptions } from '../../src/client-options'
+import { RPCResponse } from '../../src/rpc/rpc-response'
+
+describe('RPC response', () => {
+  let services: any
+  let rpcResponse: RPCResponse
+  const name = 'myRPC'
+  const correlationId = 'correlationId'
+
+  beforeEach(() => {
+    services = getServicesMock()
+    rpcResponse = new RPCResponse({
+        topic: TOPIC.RPC,
+        action: RPC_ACTIONS.REQUEST,
+        name,
+        correlationId
+      },
+      DefaultOptions,
+      services
+    )
+    rpcResponse.autoAccept = false
+  })
+
+  afterEach(() => {
+    services.connectionMock.verify()
+  })
+
+  it('doesn\'t accept automatically when autoAccept == false', () => {
+    services.connectionMock
```

Comment:
this does not make sense

Reply:
unless the ack is async, in which case it does, but then the test needs to be async or it will obviously never be called